src/pages/medicationPages/FormPage.js

```diff
@@ -63,13 +63,13 @@ export default function FormPage({
         Medikamente:
         <TextareaWithPlaceholderWrapper>
           <Textarea
-            name="meds"
+            name="inputValue"
```

Comment:
it has to be generic because I use it for different forms and since a note is not really an observation I changed it to this. But maybe I can find a better name